### PR TITLE
Added error handling and response formatting to bankid cancel lambda

### PR DESCRIPTION
**Pull request by TeddyAndersson**

### CODE REVIEW

File: services/bankid-api/lambdas/cancel.js [hunk 1]
```diff
@@ -1,31 +1,49 @@
-import { failure, success } from "../../../libs/response";
+import { snakeCaseKeys } from 'snakecase-keys';
+import to from "await-to-js";
+import { throwError } from "@helsingborg-stad/npm-api-error-handling";
 import params from "../../../libs/params";
+import * as response from "../../../libs/response";
 import * as request from "../../../libs/request";
-import * as bankId from '../helpers/bankId';
+import * as bankId from "../helpers/bankId";
```
Comment (dannil76) [inline services/bankid-api/lambdas/cancel.js]: Singel...

File: services/bankid-api/lambdas/cancel.js [hunk 2]
```diff
@@ -1,31 +1,49 @@
-import { failure, success } from "../../../libs/response";
+import { snakeCaseKeys } from 'snakecase-keys';
+import to from "await-to-js";
```
Comment (dannil76) [inline services/bankid-api/lambdas/cancel.js]: Single

File: services/bankid-api/lambdas/cancel.js [hunk 3]
```diff
@@ -1,31 +1,49 @@
...
+import { throwError } from "@helsingborg-stad/npm-api-error-handling";
```
Comment (dannil76) [inline services/bankid-api/lambdas/cancel.js]: Single

File: services/bankid-api/lambdas/cancel.js [hunk 4]
```diff
@@ -1,31 +1,49 @@
...
 import params from "../../../libs/params";
+import * as response from "../../../libs/response";
```
Comment (dannil76) [inline services/bankid-api/lambdas/cancel.js]: single

File: services/bankid-api/lambdas/cancel.js [hunk 5]
```diff
@@ -1,31 +1,49 @@
...
+async function sendBankIdCancelRequest(params, payload) {
+  let err, bankIdClientResponse, bankIdCancelResponse;
+
+  [err, bankIdClientResponse] = await to(bankId.client(params));
+  if (!bankIdClientResponse) throwError(503);
+
+  [err, bankIdCancelResponse] = await to(
+    request.call(
+      bankIdCancelResponse,
+      "post",
```
Comment (dannil76) [inline services/bankid-api/lambdas/cancel.js]: single

File: services/bankid-api/lambdas/cancel.js (hunk 1 above)
Comment (dannil76) [reply services/bankid-api/lambdas/cancel.js]: I'll fix this

File: services/bankid-api/lambdas/collect.js [hunk 6]
```diff
@@ -1,5 +1,6 @@
 import * as response from '../../../libs/response';
 import params from "../../../libs/params";
```
Comment (dannil76) [inline services/bankid-api/lambdas/collect.js]: Missed this one?

Comment (TeddyAndersson) [reply services/bankid-api/lambdas/collect.js]: Done

**Pull request merged**